index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <!-- this is the navigation header -->
+    <header class="navigation">
+        <div class="nav-logo">
+            <a href="">WELCOME</a>
+        </div>
+        <nav class="toggle"><button><i class="fa-solid fa-bars"></i></button></nav>
+    </header>
+
+    <!-- this is the section part -->
+
+    <section class="first-section">
+
+        <div class="section2">
+            <h1>I am Ekene, <br> glad to see you!</h1>
```

Comment:
- [ ] (OPTIONAL) It would be great if you could apply the same font-weight to your title so that it matches the title in the Figma template 👍